SearchInTwoDMatrix.java

```diff
@@ -0,0 +1,39 @@
+// Brute force TC:O(m*n)
+// Find row number and then binary search TC:O(m + log n)
+// Binary search on row and then binary search on column TC: O(log m + log n)
+
+// Time Complexity :O(1)
+// Space Complexity :O(logmn)
+// Did this code successfully run on Leetcode :yes
+// Any problem you faced while coding this :no
+public class SearchInTwoDMatrix {
+
+    public boolean searchMatrix(int[][] matrix, int target) 
+    {
+        if(matrix == null || matrix[0].length == null)
```

Comment:
@pkariya1 you did not notice the second part of || condition is not correct.

Reply:
@shrutikamtekar Please don't commit silly mistakes going forward.